pointcloud_topic_name : /points2
base_frame_name : base_footprint
# base_frame_name : azure_camera_base
publish_cloud_detection_range : true
publish_cloud_object : true
publish_pose_array : true
use_tf : true

passthrough_x_min : 0.0
passthrough_x_max : 1.5
passthrough_y_min : -0.5
passthrough_y_max : 0.5
passthrough_z_min : 0.5
passthrough_z_max : 1.5

use_voxel : false
leaf_size : 0.01

cluster_tolerance : 0.03
min_cluster_point_size : 200
max_cluster_point_size : 10000

threshold_distance : 0.03
probability : 0.95

object_size_x_min : 0.00
object_size_x_max : 0.40
object_size_y_min : 0.00
object_size_y_max : 0.40
object_size_z_min : 0.00
object_size_z_max : 0.40

object_centroid_offset_x : 0.0
object_centroid_offset_y : 0.0
object_centroid_offset_z : 0.0